### PR TITLE
separate each model as a new step in gh actions ...

### PR DESCRIPTION
**Pull request by bl-young**
… to enable testing of all models using `fail-fast: false`

requires new function `printValidationResults` for brevity

### CODE REVIEW

**Comment by bl-young** [comment]
Note this update to gh actions highlights #190 but shows other models still passing.

**Comment by bl-young** [comment]
@MoLi7 mostly just wanted to flag the new function here before bringing in to develop. It mimics your .rmd validation scripts.

**Comment by MoLi7** [comment]
The separation makes sense to me - thanks for setting it up.

File: R/ValidateModel.R [hunk 1]
```diff
@@ -269,3 +269,42 @@ checkNamesandOrdering <- function(n1, n2, note) {
...
+  q_x_val <- compareCommodityOutputXMarketShareandIndustryOutputwithCPITransformation(model, tolerance = 0.01)
+  print(paste("Number of flow totals by commodity passing:",q_x_val$N_Pass))
+  print(paste("Number of flow totals by commodity failing:",q_x_val$N_Fail))
+  print(paste("Sectors with flow totals failing:", paste(unique(q_x_val$Failure$rownames), collapse = ", ")))
+  
+  print("Validate that commodity output equals to domestic use plus production demand")
+  q_val <- compareCommodityOutputandDomesticUseplusProductionDemand(model, tolerance = 0.01)
+  print(paste("Number of flow totals by commodity passing:",q_val$N_Pass))
+  print(paste("Number of flow totals by commodity failing:",q_val$N_Fail))
+  print(paste("Sectors with flow totals failing:", paste(unique(q_val$Failure$rownames), collapse = ", ")))
```
Comment (MoLi7) [inline R/ValidateModel.R:309]: Per our discussion in Issue https://github.com/USEPA/useeior/issues/190, make this validation optional?

```suggestion
if (model$specs$CommodityorIndustryType=="Commodity") {
  print("Validate that commodity output equals to domestic use plus production demand")
  q_val <- compareCommodityOutputandDomesticUseplusProductionDemand(model, tolerance = 0.01)
  print(paste("Number of flow totals by commodity passing:",q_val$N_Pass))
  print(paste("Number of flow totals by commodity failing:",q_val$N_Fail))
  print(paste("Sectors with flow totals failing:", paste(unique(q_val$Failure$rownames), collapse = ", ")))
}
```

**Pull request merged**